hebi_a-2084-01_moveit_config/package.xml

```diff
@@ -3,7 +3,7 @@
   <name>hebi_a-2084-01_moveit_config</name>
   <version>0.3.0</version>
   <description>
-     An automatically generated package with all the configuration and launch files for using the A-2084-01 with the MoveIt! Motion Planning Framework
+     An automatically generated package with all the configuration and launch files for using the a-2084-01 with the MoveIt! Motion Planning Framework
```

Comment:
Again, potentially keep the `A-2084-01` capitalized in the text descriptions?  I'd welcome a second opinion from the mechanical team here.